wolfcrypt/test/test.c

```diff
@@ -471,6 +471,10 @@ int wolfcrypt_test(void* args)
 {
     int ret;
 
+    if ((ret = wolfCrypt_Init()) != 0) {
```

Comment:
This is already done in calling `main` function. The calling application should ensure wolfCrypt_Init or wolfSSL_Init are called, but if you do put it here you also need the wolfCrypt_Cleanup call.